### PR TITLE
Filled "how google work" and added one question

### PR DESCRIPTION
**Pull request by gustavo-godoy**
Hi! Hopefully I'm being useful!

Included a simple suggested answer on how searching on google works, as it was empty.
Included a question and suggested answer on how IP works (there was one for DNS, but not for IP)

### CODE REVIEW

File: web_development_101/the_basics/how_does_the_web_work.md [hunk 1]
```diff
@@ -42,16 +42,27 @@ At the end of this lesson, you should be able to do the following:
...
+  <summary>In your own words, explain what happens when you run a search on google.com.</summary>
   <ul><ul>
-    <li>DNS Servers are the "phone book" of the internet. They translate domain names into IP addresses.</li>
+    <li>Google receives the request, look for it in their databases and send back the results</li>
   </ul></ul>
  </details>
  
  <details>
-  <summary>In your own words, explain what happens when you run a search on google.com.</summary>
+  <summary>Explain what IP address is</summary>
```
Comment (marvingay) [inline web_development_101/the_basics/how_does_the_web_work.md:53]: Should be -> Explain what IP addresses are

File: web_development_101/the_basics/how_does_the_web_work.md [hunk 2]
```diff
@@ -42,16 +42,27 @@ At the end of this lesson, you should be able to do the following:
...
+  <ul><ul>
+    <li>IP adrress is a number that identifies a destination in a network.</li>
```
Comment (marvingay) [inline web_development_101/the_basics/how_does_the_web_work.md:55]: An IP Address is technically not a number, it needs a more appropriate descriptor.

**Comment by gustavo-godoy** [comment]
Thanks for you suggestions! You are right, I've reworded the IP addresses section and fixed the ordering.

File: web_development_101/the_basics/how_does_the_web_work.md (hunk 2 above)
Comment (gustavo-godoy) [reply web_development_101/the_basics/how_does_the_web_work.md]: Whoops, I've accidentally marked that as "resolved" before fixing it, sorry.

**Comment by marvingay** [comment]
@gustavo-godoy Thanks again!

**Pull request merged**